azurerm/internal/services/keyvault/key_vault_resource.go

```diff
@@ -195,6 +195,42 @@ func resourceArmKeyVault() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"private_endpoint_connection": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"private_endpoint_id": {
```

Comment:
this can be moved to a top-level field, I guess, but it's not immediately useful here since this won't be available until a Private Endpoint is connected via a separate resource - so ultimately this doesn't appear to warrant exposing on this resource?